src/styles/blocks/watch.scss

```
@@ -0,0 +1,63 @@
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.watch {
+  border-radius: 50%;
+  @include watch-base($watch-size);
```

хороша назва функції або міксіну це коли інщому розробнику не потрібно в неї заходи що б зрозуміти що вона робить а просто довіряти їй
```suggestion
  @include square($watch-size);
```